install_config/storage_examples/gluster_backed_registry.adoc

```diff
@@ -0,0 +1,137 @@
+= Backing Docker Registry with GlusterFS Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+== Overview
+
+This guide will cover the how to attach a GlusterFS persistent volume to the Docker Registry.
+It is assumed that the docker registry service has already been started and the gluster volume has been created.
+
+
+=== Prerequisites
+* The link:../install/docker_registry.html#deploy-registry[docker-registry] has been deployed *without*
+configuring storage.
+* A gluster volume exists and *glusterfs-fuse* is installed on all hosts.
```

Comment:
"gluster-fuse" package...
                      ^^^^
on all scheduleable hosts (not sure if that's a word but the point is that some hosts are masters and thus do not need glusterfs-fuse.
          ^^^